examples/recipes/multiple-data-sources/cube.js

```diff
@@ -0,0 +1,27 @@
+const PostgresDriver = require('@cubejs-backend/postgres-driver');
+
+module.exports = {
+    contextToAppId: ({ securityContext }) => 
+      `CUBEJS_APP_${securityContext.tenant}`,
+
+    driverFactory: ({ securityContext } = {}) => {
```

Comment:
It's just a cherry on top, but I think that having comments on top of every function here with concise explanation why this function is present here would help in complex cases (e.g., multitenancy).

Reply:
Added comments